src/types.ts

```diff
@@ -453,15 +457,21 @@ export interface AccountTransactionHeader {
     expiry: TransactionExpiry;
 }
 
-export interface SimpleTransfer {
+export interface SimpleTransferPayload {
     /** µGTU amount to transfer */
     amount: GtuAmount;
 
-    /** the recipient of the transfer*/
+    /** the recipient of the transfer */
     toAddress: AccountAddress;
 }
 
-export type AccountTransactionPayload = SimpleTransfer;
+export interface SimpleTransferWithMemoPayload extends SimpleTransferPayload {
+    /** The provided memo of the transaction,
+       * expected to be a HEX string representing the bytes of the memo */
+    memo: string;
```

Comment:
I think the typing should be a `Buffer` and not the visual representation of it (a hex string).